lib/selftest/src/Obelisk/SelfTest.hs

```diff
@@ -243,6 +243,8 @@ main' isVerbose httpManager obeliskRepoReadOnly = withInitCache $ \initCache ->
       uu <- update
       assertRevEQ u uu
 
+    it "can run 'ob doc'" $ inTmpObInit $ \_ -> runOb ["doc", "reflex"]
```

Comment:
Does this actually fail without the fix?

Reply:
Yep

Reply:
```
  src/Obelisk/SelfTest.hs:246:5: 
  1) blank initialized project can run 'ob doc'
       uncaught exception: ReThrownException
       
       Ran commands: 
       cd /tmp/test λ-61f9295e548d4478
       /nix/store/3kqc2wmvf1jkqb2jmcm7rvd9lf4345ra-coreutils-8.31/bin/cp -rT '/tmp/init Cache λ-697a7e209a58e822' '/tmp/test λ-61f9295e548d4478'
       which /nix/store/3kqc2wmvf1jkqb2jmcm7rvd9lf4345ra-coreutils-8.31/bin/cp
       ob doc reflex
       which ob
       cd /nix/store/91k5g8wvnb4bk75m9nj3rqpdvzw60ix3-impl
       
       Exception: error running: ob doc reflex
       exit status: 2
       stderr: Alert: No valid, buildable packages found
```